rest/Cargo.toml

```diff
@@ -39,10 +39,10 @@ paperclip = { version = "0.5.0", default-features = false, optional = true }
 mayastor-macros = { path = "../mayastor-macros" }
 http = "0.2.3"
 tinytemplate = { version = "1.2" }
+rpc = { path = "../rpc" }
```

Comment:
why does the rest need to depend on the rpc?

Reply:
It doesn't - this was my mistake. Removed it.